nifi-pulsar-processors/src/main/java/org/apache/nifi/processors/pulsar/pubsub/ConsumePulsarRecord.java

```diff
@@ -356,7 +356,7 @@ public Object call() throws Exception {
     	}
     }
     
-    private void handleFailures(ProcessSession session, BlockingQueue<Message<byte[]>> parseFailures, byte[] demarcator) {
+    private void handleFailures(ProcessSession session, Queue<Message<byte[]>> parseFailures, byte[] demarcator) {
```

Comment:
Why do you no longer use a BlockingQueue?

Reply:
See above.